### DISCUSSION TITLE
Install Simnibs 4.1.0 on Linux headless server

### DISCUSSION THREAD

**Discussion by michefabb** [simnibs/simnibs#475] [6. Getting started] [answered]
Dear SimNIBS experts,

I tried to install SimNIBS on a Linux headless server using Conda, following instructions at [https://simnibs.github.io/simnibs/build/html/installation/conda.html](https://simnibs.github.io/simnibs/build/html/installation/conda.html) , however it looks this result in an error.
In particular it looks that the packages in the .yml file
  - pyqt=5.9.2
  - mkl=2022.0.1
are problematic and no mkl exists for linux aarch64 and osx-arm64...however when I installed it on my local Mac using the .pkg it was installed in the conda env simnibs_env without problem...could you kindly help? Can mkl=2022.0.1 be replaced with nomkl=3.0 ?

Thanks, regards,

Mic

**Comment by TorgeW** [comment, marked as answer]
pyqt is only needed for the GUI, so leaving it out should not cause any issues. 

SimNIBS doesn't support arm, only apple silicon will be supported in the near future. 
Feel free to try replacing mkl with nomkl but we can not offer support on this.

**Reply by michefabb** [reply]
Thanks @TorgeW !
